tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -673,13 +745,25 @@ public TestSetup apply(Long subscribers) throws Throwable {
           // sub1 now has 18 pending
           // sub2 has 0 pending
 
-          final T z = sendNextTFromUpstream();
-          expectNextElement(sub1, z);
-          sub2.expectNone(); // since sub2 hasn't requested anything yet
+          if (doesCoordinatedEmission()) {
+            sub2.expectNone(); // since sub2 hasn't requested anything yet
+
+            sub2.request(1);
 
-          sub2.request(1);
-          expectNextElement(sub2, z);
+            final T z = sendNextTFromUpstream();
+            expectNextElement(sub1, z);
+            expectNextElement(sub2, z);
+          } else {
+            final T z = sendNextTFromUpstream();
+            expectNextElement(sub1, z,
+                    "If the Processor coordinates requests/emissions when having multiple Subscribers"
+                            + " at once, please override doesCoordinatedEmission() in this "
```

Comment:
"please override doesCoordinatedEmission() to return `true` in this"

Reply:
Updated.